frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -0,0 +1,234 @@
+<template lang='pug'>
+modal-base-template(ref='modal' :fullscreen='true' class='has-background')
+  .c-payment-form(v-if='!donePayment')
+    .c-header
+      i18n.is-title-2.c-title(tag='h2') Record payments
+
+    .c-content
+      form.card.c-card(
+        @submit.prevent='submit'
+        novalidate='true'
+      )
+        i18n.has-text-bold.c-title(tag='h3') Who did you send money to?
+        payments-list(
+          :titles='tableTitles'
+          :payments='fakeStore.paymentsDistribution'
+          paymentsType='edit'
+        )
+
+        .c-footer
+          .c-footer-info
+            i18n.has-text-bold(
+              tag='h4'
+              :args='{ optional: `<span class="has-text-small has-text-1 has-text-normal">(${L("optional")})</span>`}'
+            ) Add a note {optional}
+
+            i18n.has-text-small.has-text-1(
+              tag='p'
+            ) Leave a message to the group members selected above.
+          .c-footer-action
+            input.switch#showComment(
+              type='checkbox'
+              name='displayComment'
+              @change='displayComment = !displayComment'
+            )
+            i18n.sr-only(tag='label' for='displayComment') Toggle comment box
+
+        transition(name='slidedown')
+          label.field(v-if='displayComment')
+            i18n.sr-only.label Leave a message
+            textarea.c-comment(
+              rows='4'
+            )
+
+        .buttons
+          i18n.is-outlined(
+            tag='button'
```

Comment:
Inside a `<form>`, all buttons that are not a "submit" should explicitly have a `type="button"`, otherwise when you press Enter while focused in another button (example: Cancel or Add a note Switch), it will always submit the form instead. Another approach (simpler and better in my opinion) is to change form submit to just `@submit.prevent=''` (do nothing). That way the user has to explicitly click "Register payments".